addons/analytic/models/analytic_plan.py

```diff
@@ -44,6 +44,10 @@ def _default_color(self):
         'parent_id',
         string="Childrens",
     )
+    all_children_ids = fields.One2many(
```

Comment:
I was thinking of not having the new field and computing the value directly in `_compute_all_analytic_account_count`, since we don't need it elsewhere.
Just to avoid needing an exception :shrug:

Reply:
oh right, I thought the problem was just the fact it was recursive, sorry